contentcuration/contentcuration/frontend/shared/views/__tests__/studioBanner.spec.js

```diff
@@ -0,0 +1,53 @@
+import { render, screen } from '@testing-library/vue';
+import VueRouter from 'vue-router';
+import StudioBanner from '../StudioBanner.vue';
+
+// Mock the Kolibri design system composable
+const mockSendPoliteMessage = jest.fn();
+jest.mock('kolibri-design-system/lib/composables/useKLiveRegion', () => ({
+  __esModule: true,
+  default: () => ({
+    sendPoliteMessage: mockSendPoliteMessage,
+  }),
+}));
+
+const sampleErrorMessage = 'This is an error message';
+const sampleInfoMessage = 'This is an info message';
+
+// Helper function to render the component with the provided props
+const renderComponent = (props = {}) =>
+  render(StudioBanner, {
+    props: {
+      error: false,
+      errorText: '',
+      ...props,
+    },
+    routes: new VueRouter(),
+  });
+
+describe('StudioBanner', () => {
+  beforeEach(() => {
+    mockSendPoliteMessage.mockClear();
+  });
+
+  test('smoke test - with default props', () => {
```

Comment:
renaming this to something like `renders  banner with  error text` to describe behavior more explicitly.